docs/general/performance.md

```diff
@@ -0,0 +1,8 @@
+# Performance
```

Comment:
Did we get rid of the old docs for this? I thought we already had a section on this topic

Reply:
I'm not aware of any docs that ever existed for this. Happy to combine if anyone is aware of anything like that

Reply:
I was mistaken. Couldn’t find anything